main.go

```diff
@@ -72,6 +77,41 @@ func dataVisHandler(w http.ResponseWriter, req *http.Request) {
 }
 
 func apiHandler(w http.ResponseWriter, req *http.Request) {
+	log.DebugR(req, "returning api help text", log.Data{
+		"host": req.Host,
+		"path": req.URL.Path,
+	})
 	w.WriteHeader(410)
 	w.Write([]byte(apiResponse))
 }
+
+func visualAssetHandler(w http.ResponseWriter, req *http.Request) {
+	dest := "https://static.ons.gov.uk/visual/" + mux.Vars(req)["uri"]
+	log.DebugR(req, "redirecting visual.ons.gov.uk wp-content", log.Data{
+		"host": req.Host,
+		"path": req.URL.Path,
+		"dest": dest,
+	})
+	w.Header().Set("Location", dest)
+	w.WriteHeader(redir)
+}
+
+func visualArticleHandler(w http.ResponseWriter, req *http.Request) {
+	article := mux.Vars(req)["article"]
+	uri := mux.Vars(req)["uri"]
+	log.DebugR(req, "redirecting visual.ons.gov.uk", log.Data{
+		"article": article,
+		"uri":     uri,
+		"host":    req.Host,
+		"path":    req.URL.Path,
+	})
+
+	if dest, ok := visualRedirects[article]; ok {
+		w.Header().Set("Location", dest)
+		w.WriteHeader(redir)
+		return
+	}
+
```

Comment:
might be worth adding a log message here to say defaulting to National archives redirect.

Reply:
done 👍